src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.Reflection.cs

```diff
@@ -14,11 +14,9 @@ namespace Uno.UI.SourceGenerators.XamlGenerator
 	internal partial class XamlFileGenerator
 	{
 		private Func<string, INamedTypeSymbol?>? _findType;
-		private Func<XamlType, INamedTypeSymbol?>? _findTypeByXamlType;
+		private Func<XamlType, bool, INamedTypeSymbol?>? _findTypeByXamlType;
```

Comment:
It looks like conflicts were incorrectly resolved here. The removal of that bool was intentional.

Reply:
Ah, sorry about that!

Reply:
No problem!
I anticipated this could happen, and it was caught be build failures anyway :)

Reply:
I have adjusted this, can you verify it is correct now please?

Reply:
It looks good now. Thanks @MartinZikmund !